book.js

```diff
@@ -1,47 +1,56 @@
 let books = JSON.parse(localStorage.getItem('books')) || [];
-const bookListDiv = document.getElementById("book-list");
-const addBookForm = document.getElementById("add-book-form");
 
-// Define the functions to add and remove books
 function addBook(title, author) {
-  books.push({ title: title, author: author });
+  books.push({ title, author });
```

Comment:
[OPTIONAL]
- [ ] It would be great to assign each book a unique identifier id wich may serve you to delete it as well.
hint: You may consider saying:
```js
books.push({title, author, id: Date.now})
```